app/components/utils/radio-dropdown-combo-select/template.hbs

```diff
@@ -0,0 +1,41 @@
+<div class="vlc-radio-inline">
+  {{#each this.radioOptions as |opt index|}}
+    {{#let (concat this.radioName "-" index) as |radioId|}}
+      <label class="vl-radio" for={{radioId}}>
+        <Input
+          class="vl-radio__toggle"
+          id={{radioId}}
+          name={{this.radioName}}
+          @type="radio"
+          value={{get opt this.labelKey}}
+          checked={{eq opt this.selectedOption}}
+          {{on "input" (fn this.select opt)}}
+        />
+        <div class="vl-radio__label">{{get opt this.labelKey}}</div>
+      </label>
+    {{/let}}
+  {{/each}}
+  {{#let (concat this.radioName "-" "others") as |radioId|}}
+    <label class="vl-radio" for={{radioId}}>
+      <Input
+        class="vl-radio__toggle"
+        id={{radioId}}
+        name={{this.radioName}}
+        @type="radio"
+        value="Andere"
+        checked={{this.selectedIsInOthers}}
+        {{on "input" this.showOthers}}
+      />
+      <div class="vl-radio__label">Andere</div>
```

Comment:
Linter error `no-bare-strings`

Reply:
heb een lijntje bijgemaakt in mapping file :-)

Reply:
Er zit geen change in nl-be.json, dus hij vind die vertaalsleutel niet